inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/InAppMessaging.kt

```diff
@@ -105,12 +105,19 @@ abstract class InAppMessaging internal constructor() {
         /**
          * Configures the In-App Messaging SDK.
          *
+         * @param context Context object.
+         * @param subscriptionKey An optional subscription key. Default is the value set in your app's AndroidManifest.
+         * @param configUrl An optional config URL. Default is the value set in your app's AndroidManifest.
+         *
          * @return `true` if configuration is successful, and `false` otherwise.
          */
         @SuppressWarnings("TooGenericExceptionCaught")
-        fun configure(context: Context): Boolean {
+        fun configure(context: Context, subscriptionKey: String? = null, configUrl: String? = null): Boolean {
```

Comment:
Please check if this is breaking change in Java app.

Reply:
Confirmed not a breaking change in a Java app, but needs the `JvmOverloads` annotation to be able to call the extended API
Added